### PR TITLE
Changes made by ammar

### PR DESCRIPTION
**Pull request by ag-mahoon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,8 +2,9 @@
   "name": "immersive",
   "version": "1.0.0",
   "description": "Assignment to create CRUD operations",
-  "main": "server.ts",
+  "main": "server.js",
```
Comment (ag-mahoon) [inline package.json:5]: what was the reason to change this file?

File: package.json [hunk 2]
```diff
@@ -2,8 +2,9 @@
...
   "scripts": {
+    "practice": "node server.js",
```
Comment (ag-mahoon) [inline package.json:7]: Nodemon is already available in the next line...

Comment (MrHusnain) [reply package.json:7]: ok

File: package.json (hunk 1 above)
Comment (ag-mahoon) [reply package.json:5]: I've have filxed isue.

**Pull request merged**